news/2025/2025-09-30-moscow-osu-event-2025-recap.md

```diff
@@ -0,0 +1,141 @@
+
+### osu!mania 4K
+
+The osu!mania 4K tournament featured ::{ flag=RU }:: [-Oni-Chan-](https://osu.ppy.sh/users/10036005), ::{ flag=RU }:: [Dystire](https://osu.ppy.sh/users/11585213), ::{ flag=RU }:: [kloofhi](https://osu.ppy.sh/users/29627572), and ::{ flag=RU }:: [VikaGamer228](https://osu.ppy.sh/users/20049082), two of which were in the [osu!mania 4K World Cup 2024](/wiki/Tournaments/MWC/2024_4K)!
+
+| Player A |  |  | Player B | VOD link |
+| --: | :-: | :-: | :-- | :-: |
+| **[Dystire](https://osu.ppy.sh/users/11585213)** ::{ flag=RU }:: | **4** | 0 | ::{ flag=RU }:: [VikaGamer228](https://osu.ppy.sh/users/20049082) | [#1](https://youtube.com/watch?v=LzItzsD4nH0&t=3745) |
+| [kloofhi](https://osu.ppy.sh/users/29627572) ::{ flag=RU }:: | 2 | **4** | ::{ flag=RU }:: **[-Oni-Chan-](https://osu.ppy.sh/users/10036005)** | [#1](https://youtube.com/watch?v=LzItzsD4nH0&t=6429) |
+| **[Dystire](https://osu.ppy.sh/users/11585213)** ::{ flag=RU }:: | **5** | 1 | ::{ flag=RU }:: [-Oni-Chan-](https://osu.ppy.sh/users/10036005) | [#1](https://youtube.com/watch?v=fIpS-LFpB7w&t=2811) |
+
+### osu!mania 7K
+
+The osu!mania 7K tournament featured ::{ flag=RU }:: [karayss](https://osu.ppy.sh/users/7156634), ::{ flag=RU }:: [PhobosX](https://osu.ppy.sh/users/2570019), ::{ flag=RU }:: [timebuka](https://osu.ppy.sh/users/6857841), and ::{ flag=RU }:: [X_Devil](https://osu.ppy.sh/users/6694908); two of them were in the [osu!mania 7K World Cup 2025](/wiki/Tournaments/MWC/2025_7K)!!
+
+| Player A |  |  | Player B | VOD link |
+| --: | :-: | :-: | :-- | :-: |
+| [PhobosX](https://osu.ppy.sh/users/2570019) ::{ flag=RU }:: | 0 | **4** | ::{ flag=RU }:: **[karayss](https://osu.ppy.sh/users/7156634)** | [#1](https://youtube.com/watch?v=xatn-BXpMho&t=19017) |
+| [X_Devil](https://osu.ppy.sh/users/6694908) ::{ flag=RU }:: | 3 | **4** | ::{ flag=RU }:: **[timebuka](https://osu.ppy.sh/users/6857841)** | [#1](https://youtube.com/watch?v=xatn-BXpMho&t=22857) |
+| [karayss](https://osu.ppy.sh/users/7156634) ::{ flag=RU }:: | 4 | **5** | ::{ flag=RU }:: **[timebuka](https://osu.ppy.sh/users/6857841)** | [#1](https://youtube.com/watch?v=fIpS-LFpB7w&t=13209) |
+
+*Fun fact: the osu!mania 7K tournament had more tiebreaker matches than all other game modes combined, including osu!.*
+
+---
+
+More than **450** people visited Moscow osu! Event 2025, with **51** individual contributors and staff members making sure everything goes smooth. We are very grateful to all participants and hope that next iterations will be even bigger and bring even more fun! Make sure to follow Moscow osu! Event's [Discord server](https://discord.gg/7GJvD9uyV7), [Twitter](https://x.com/moscowosuevent), [Telegram](https://t.me/moscowosuevent), or [VK](https://vk.com/moscowosuevent), so that you don't miss the next iteration!
```

Comment:
```suggestion
More than **450** people visited Moscow osu! Event 2025, with **51** individual contributors and staff members making sure everything went smoothly. We are very grateful to all participants and hope that the next iteration will be even bigger and bring even more fun! Make sure to follow Moscow osu! Event's [Discord server](https://discord.com/invite/7GJvD9uyV7), [Twitter](https://x.com/moscowosuevent), [Telegram](https://t.me/moscowosuevent), or [VK](https://vk.com/moscowosuevent), so that you don't miss the next iteration!
```